### ISSUE TITLE
extraneous margins on product list and product list items

### ISSUE THREAD

**Issue by DanielRuf** [magento/magento2#15308]
### Preconditions

1. Magento 2.2.4

### Steps to reproduce

1. open list view of products on a page
2. inspect the list `.column.main .product-items` and one product `.products-list .product-item`

### Expected result

1. there should not be any extraneous styles

### Actual result

1. there are negative and positive margins which do nothing combined

The following code would remove it:
```css
.products-list .product-item {
    padding-left: 0;
  }
  .column.main .product-items {
    margin-left: 0;
  }
```

In general I see no sense in these margins and paddings.

See https://github.com/magento/magento2/blob/c0123f3bcaf3b3878b9b6e425f92f16468aaf87c/app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less#L167-L177

https://github.com/magento/magento2/blame/c0123f3bcaf3b3878b9b6e425f92f16468aaf87c/app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less#L167-L177

https://github.com/magento/magento2/commit/08b6a7fa8b9f10b0c73ff9c4cebc08e048b8d7a3

**Comment by magento-engcom-team** [comment]
Hi @DanielRuf. Thank you for your report. 
The issue has been fixed in magento/magento2#15936 by @StevenGuapaBV in 2.2-develop branch
Related commit(s):
 - [cae12b165bc385ef2037ba12fa663f0600cf98ba](https://github.com/magento/magento2/commit/cae12b165bc385ef2037ba12fa663f0600cf98ba)

The fix will be available with the upcoming 2.2.6 release.

**Comment by VladimirZaets** [comment]
Hi @DanielRuf. Thank you for your report. 
The issue has been fixed in magento/magento2#15975 by @chirag-wagento in 2.3-develop branch
Related commit(s):
 - [eee2e7ccd7aef585e973df28719b25941e7e401e](https://github.com/magento/magento2/commit/eee2e7ccd7aef585e973df28719b25941e7e401e)

The fix will be available with the upcoming 2.3.0 release.

**Comment by sidolov** [comment]
Hi @DanielRuf. Thank you for your report. 
The issue has been fixed in magento/magento2#17379 by @gelanivishal in 2.1-develop branch
Related commit(s):
 - [ebdc7c3770a32cce2944344ecb73def07738b690](https://github.com/magento/magento2/commit/ebdc7c3770a32cce2944344ecb73def07738b690)

The fix will be available with the upcoming 2.1.15 release.
